### DISCUSSION TITLE
How to run only doctests?

### DISCUSSION THREAD

**Discussion by le-chartreux** [pytest-dev/pytest#11116] [Q&A] [answered]
Hello,

Thank you for developing pytest, that's very cool!

Is there an option inside pytest to only run doctests? 
I check them with `pytest --doctest-modules`, but it obviously runs my unit tests too. 
The only workaround I figured is to do `pytest src/ --doctest-modules`, so unit tests that are in `tests/` are not executed. 

Have a nice day :)

**Comment by twmr** [comment]
I'm also using this "workaround". I doubt that what you want is supported by pytest OOTB.

**Comment by RonnyPfannschmidt** [comment, marked as answer]
Currently there's no builtin way to select tests by Item class

**Reply by le-chartreux** [reply]
OK, thank you!
